packages/bpk-component-price-range/src/common-types.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+export const MARKER_TYPES = {
+  low: 'low',
+  typical: 'typical',
+  high: 'high',
```

Comment:
I would recommend use upper case for const variables, eg 
```
export const MARKER_TYPES = {
  LOW: 'low',
  // ...
} as const;

```

Reply:
Nice, updated, thanks